src/assignment/views/AssignmentOverview.tsx

```diff
@@ -112,13 +112,16 @@ const AssignmentOverview: FunctionComponent<AssignmentOverviewProps> = ({
 		try {
 			if (user) {
 				setCanEditAssignments(
-					await PermissionService.hasPermissions(PermissionName.EDIT_ASSIGNMENTS, user)
+					await PermissionService.hasPermissions(
+						[PermissionName.EDIT_OWN_ASSIGNMENTS, PermissionName.EDIT_ASSIGNMENTS],
```

Comment:
shouldn't this also include the new permission: EDIT_ALL_ASSIGNMENTS ?

Reply:
indeed. fixed